File: InverterString.java
public class InverterString {

  public static void main(String[] args) {
    String str = "Exemplo"; // Armazena string
    String invertida = "";

    for (int i = str.length() - 1; i >= 0; i--) { // Laço de repetição que inverterá a string
      invertida += str.charAt(i);
    }

    System.out.println(invertida);
  }
}

File: InverterString.py
def inverter_string(str):
    invertida = ''
    for i in range(len(str) - 1, -1, -1): # Laço de repetição que inverterá a string
        invertida += str[i]
    return invertida

str = "Exemplo" # Armazenar string
invertida = inverter_string(str)
print(invertida)
